relay-server/src/services/processor/span/processing.rs

```diff
@@ -413,6 +467,73 @@ fn validate(mut span: Annotated<Span>) -> Result<Annotated<Span>, anyhow::Error>
     if let Some(tags) = tags.value_mut() {
         tags.retain(|_, value| !value.value().is_empty())
     }
+    if let Some(measurements) = measurements.value_mut() {
+        measurements.retain(|_, value| match value.value() {
+            Some(v) => !v.is_empty() && !v.value.is_empty(),
+            None => false,
+        });
+    }
+    if let Some(metrics_summary) = _metrics_summary.value_mut() {
+        metrics_summary.0.retain(|_, value| match value.value() {
+            Some(v) => !v.is_empty() && v.iter().all(|v| !v.is_empty()),
```

Comment:
Are the individual fields of a metrics summary allowed to be empty? (`min`, `max`, etc.). Do we need explicit checks on those?

https://github.com/getsentry/relay/blob/27963b14672270d8cb3645e6ca6571479d2afb34/relay-event-schema/src/protocol/metrics_summary.rs#L27-L42

Reply:
They can be omitted but not set to `null`.

Reply:
The Snuba consumers will require a value though, which is not compatible with the schema. I'll fix that in there though.

Reply:
https://github.com/getsentry/snuba/pull/5428